src/node_crypto.cc

```diff
@@ -255,13 +255,53 @@ void ThrowCryptoError(Environment* env,
                       unsigned long err,  // NOLINT(runtime/int)
                       const char* default_message = nullptr) {
   HandleScope scope(env->isolate());
```

Comment:
Below is the bulk of the change. Updated `ThrowCryptoError` to get the current error state and add the stack to a new property on the exception object